tests/conftest.py

```diff
@@ -38,7 +38,7 @@ def fs():
     fs._client.close()
 
 
-@pytest.fixture
+@pytest.fixture(scope="session")
```

Comment:
I don't think session scope is the right one. I want a unique object for each test function.

Reply:
Updated scope to function.